tests/bettertransformer/testing_bettertransformer_utils.py

```diff
@@ -118,7 +119,7 @@ def test_raise_on_save(self):
         Test if the converion properly raises an error if someone tries to save the model using `save_pretrained`.
         """
         for model_id in self.all_models_to_test:
-            with self.assertWarns(UserWarning), tempfile.TemporaryDirectory() as tmpdirname:
+            with self.assertRaises(ValueError), tempfile.TemporaryDirectory() as tmpdirname:
                 hf_model = AutoModel.from_pretrained(model_id).eval()
                 bt_model = BetterTransformer.transform(hf_model, keep_original_model=False)
                 bt_model.save_pretrained(tmpdirname)
```

Comment:
No need to change, but just to let you know, you can actually even test for the error message to be the right one: https://github.com/huggingface/optimum/blob/9efb46a38c64eab31324d35d56299730612f93a8/tests/onnxruntime/test_modeling.py#L359-L364